src/script/components/MessagesList/Message/FailedToAddUsersMessage.tsx

```diff
@@ -108,25 +108,58 @@ const MessageDetails = ({failure, isMessageFocused, allUsers}: MessageDetailsPro
     </>
   );
 
+  const getText = () => {
```

Comment:
a lot of changes there, but it had to be done, because of the type, logic remains the same as it was